Formula/protozero.rb

```diff
@@ -0,0 +1,19 @@
+class Protozero < Formula
+  desc "Minimalistic protocol buffer decoder and encoder in C++"
+  homepage "https://github.com/mapbox/protozero"
+  url "https://github.com/mapbox/protozero/archive/v1.3.0.tar.gz"
+  sha256 "85f9238fa662ff06a1e364f1461846a9d377846274e7f98407307e31086cab2b"
+
+  def install
+    (include/"protozero").install Dir["include/protozero/*"]
```

Comment:
Thanks for your input @DomT4! Now that you mention it, it makes a lot of sense and I should have thought about this before making @smellman do all this work. I think I'm with you that this isn't necessarily a useful addition to Homebrew. Sorry for not realizing this earlier!